timemachine/potentials/bonded.py

```diff
@@ -79,35 +79,44 @@ def harmonic_bond(conf, params, box, bond_idxs):
     return jnp.sum(energy)
 
 
-def kahan_angle(ci, cj, ck):
+def kahan_angle(ci, cj, ck, eps):
     """
     Compute the angle given three points, i,j,k, as defined by the vector j->i, j->k
```

Comment:
Possibly document eps or point to https://github.com/proteneer/timemachine/pull/935 , https://github.com/proteneer/timemachine/pull/1271